README.adoc

```diff
@@ -138,10 +146,10 @@ include::finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.
 
 The inventory resource is a RESTful service that is served at the `inventory/systems` endpoint. 
 
-Add the [hotspot=RequestScoped file=1]`@RequestScoped` annotation on the class to indicate that this bean is
-to be initialized once for every request. In other words, the bean is instantiated when the request
-is received and destroyed when a response is sent back to the client. While this bean can
-also be application-scoped, request scope is short-lived and is therefore ideal for HTTP requests.
+Add the [hotspot=ApplicationScoped file=1]`@ApplicationScoped` annotation on the class to indicate that this bean is
+to be initialized once for the duration of the application, where the state of the bean is shared between all the requests.
+
+If you want this bean to be initialized once for every request, you can add the `@RequestScoped` annotation on the class instead. In other words, the bean is instantiated when the request is received and destroyed when a response is sent back to the client. A request scope is short-lived.
```

Comment:
Change
"If you want this bean to be initialized once for every request, you can add the `@RequestScoped` annotation on the class instead. In other words, the bean is instantiated when the request is received and destroyed when a response is sent back to the client. A request scope is short-lived."
To
"If you want this bean to be initialized once for every request, you can annotate the class with the `@RequestScoped` annotation instead. With the `@RequestScoped` annotation, the bean is instantiated when the request is received and destroyed when a response is sent back to the client. A request scope is short-lived."